src/App.tsx

```diff
@@ -1,35 +1,56 @@
 import React from 'react';
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
+
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
 
-type State = {};
+type State = {
+  inputValue: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    inputValue: '',
+  };
+
+  changeInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    this.setState({ inputValue: value });
+  };
+
+  getFilteredMovies = (inputValue: string) => {
+    return (moviesFromServer.filter(movie => movie.title.toLowerCase()
+      .includes(inputValue.toLowerCase())
+      || movie.description.toLowerCase().includes(inputValue.toLowerCase())) || null);
```

Comment:
- no need to do inputValue.toLowerCase() on every iteration, do it before filtering;
- would null be ever returned? 
- format it, now it's difficult to read